network/proto/identity.proto

```diff
@@ -22,7 +22,9 @@ message Status {
 
     int64 chain = 3;
 
-    string genesis = 4;
+    bool temporaryDial=4;
+
+    string genesis = 5;
```

Comment:
The whole point of using proto files is that they are backwards compatible.
Changing the number of the field in proto files essentially breaks it:

Please refer to the [official proto docs](https://developers.google.com/protocol-buffers/docs/proto#assigning_field_numbers) to understand why this is important

Adding new fields should involve increasing the number, not _reusing_ the old number from another field

Reply:
Commit-[link](https://github.com/0xPolygon/polygon-edge/pull/343/commits/a0959b87ebc593e502365bd91a3bc1b6011e7a4b)